lib/repos.js

```diff
@@ -102,32 +196,167 @@ Repos.prototype.read = function (callback) {
   }, []);
 
   async.map(lookups, function (obj, cb) {
```

Comment:
🌎 GLOBAL: For consistency with other function signatures in this library (e.g. `_checkAndHandleError`), consider renaming `cb` -> `callback`.